.env

```diff
@@ -0,0 +1,9 @@
+DOCKER_TAG=latest
+ETH_RPC_URL=ENC[AES256_GCM,data:PCD1HqzgS+g7XxhsZ2dxsOrm6F9NfoS3vyheLqba/syLmsmHrLaiA4jOc2JdK348ws9jM2LAuSRq1O1kyuuYXa22zQ==,iv:aUkOvK9APf0kkv9Ls6ezfhU+GvBWWsyQrgrJOeyldOk=,tag:Vofdq8gq10KSkgFH1QNT8A==,type:str]
+sops_gcp_kms__list_0__map_resource_id=projects/radicle-services/locations/global/keyRings/sops/cryptoKeys/sops-key
+sops_gcp_kms__list_0__map_created_at=2021-09-07T14:22:16Z
+sops_mac=ENC[AES256_GCM,data:fgign6WX9t2RrWTomgiDPK4uofvk55h0kBUltP4kOqRR1jQoswnOWIt4IQJa5FkKcVcTZQA6pglhu3q25DOLfsFZIKWi8ljnfvFadOKuqFaGwf/gDuDr96m/tKBzTtp8yiRwWt/yBbYuEQ10154dTkMjhVwac0rGu1S0x6ikCng=,iv:X3H4f80bU2l7UOBVRKQumoD4qzwMDeZHIw4RV6bajJw=,tag:BUQPSjV5wIApng16ZQVtmg==,type:str]
+sops_lastmodified=2021-09-07T14:22:17Z
+sops_encrypted_regex=ETH_RPC_URL
+sops_gcp_kms__list_0__map_enc=CiQAw9HjBmWd2O5n4q9FpEf+NR1rMlYgUP4HPQsmW2GVDBh8EY8SSQCYOiXngEsdj+/hKtvAQ/gmak22KjJEz4ePtvyY7X3CZFT4ofGmZYkdeI5Bfwxhid3kISattILdxFf546hnXNZHwBMfpF01EVM=
+sops_version=3.7.1
```

Comment:
Interesting, though this will only work for us :thinking: - I wonder if it belongs here.

Reply:
New keys (pgp, gcp kms, aws kms, age) can be added any time, so it isn't restricted only to people having access to that specific gcp project.

Reply:
Ok you'll have to explain!